plugin/pkg/scheduler/factory/factory.go

```diff
@@ -468,8 +539,15 @@ func (c *configFactory) addPodToCache(obj interface{}) {
 	}
 
 	c.podQueue.AssignedPodAdded(pod)
+
 	// NOTE: Updating equivalence cache of addPodToCache has been
 	// handled optimistically in InvalidateCachedPredicateItemForPodAdd.
+
+	if c.volumeBinder != nil {
```

Comment:
This is not needed. You have code to delete the bindings in the delete event handler for unassigned pods.